skunkworks_crow/build.gradle

```diff
@@ -70,4 +70,7 @@ dependencies {
 
     //MPAndroidChart
     implementation 'com.github.PhilJay:MPAndroidChart:v3.0.3'
+
+    //Loading Animations
+    implementation 'com.github.ybq:Android-SpinKit:1.2.0'
```

Comment:
I would say not to use any animation library till we don't have a proper need or requirement for it. Even in ODK-Collect we don't have any animation library so let's make it consistent everywhere.

Reply:
Sure!